libr/core/cmd_anal.c

```diff
@@ -6614,29 +6614,37 @@ static void cmd_anal_esil(RCore *core, const char *input) {
 		}
 		break;
 	case 'A': // "aeA"
-		if (input[1] == '?') {
-			r_core_cmd_help (core, help_msg_aea);
-		} else if (input[1] == 'r') {
-			cmd_aea (core, 1 + (1<<1), core->offset, r_num_math (core->num, input+2));
-		} else if (input[1] == 'w') {
-			cmd_aea (core, 1 + (1<<2), core->offset, r_num_math (core->num, input+2));
-		} else if (input[1] == 'n') {
-			cmd_aea (core, 1 + (1<<3), core->offset, r_num_math (core->num, input+2));
-		} else if (input[1] == 'j') {
-			cmd_aea (core, 1 + (1<<4), core->offset, r_num_math (core->num, input+2));
-		} else if (input[1] == '*') {
-			cmd_aea (core, 1 + (1<<5), core->offset, r_num_math (core->num, input+2));
-		} else if (input[1] == 'f') {
-			RAnalFunction *fcn = r_anal_get_fcn_in (core->anal, core->offset, -1);
-			if (fcn) {
-				cmd_aea (core, 1, r_anal_function_min_addr (fcn), r_anal_function_linear_size (fcn));
+		switch (input[1]){
+			case '?':
```

Comment:
case must be indented at the same level as the switch statement

Reply:
fixed